src/main/java/org/gridsuite/directory/server/repository/DirectoryElementEntity.java

```diff
@@ -68,11 +68,16 @@ public DirectoryElementEntity update(@NonNull ElementAttributes newElementAttrib
             this.isPrivate = newElementAttributes.getAccessRights().isPrivate();
         }
 
+        if (Objects.nonNull(newElementAttributes.getDescription())) {
```

Comment:
You need to use **StringUtils.isNotBlank**  to control whitespace and empty space.

Reply:
Done